### DISCUSSION TITLE
Installation instructions on MacOS M1? Not sure what I am doing wrong.

### DISCUSSION THREAD

**Discussion by HumbledBezalel** [bitcraze/discussions#476] [Q&A]
I was trying to install the cf client on my computer but the commands in the instructions are not clear to me, seemed not to be working on my M1 Macbook pro?  I do not see any application that I can open, is everything thru command line? Can someone check the instructions for Macbook M1 and let me know if they make sense.

Are the instructions on documentation for Mac M1 setup still viable?

**Comment by ataffanel** [comment]
This is correct that everything is intended to work in a command line. There is likely a way to add a shortcut in the system but this is not something I have experimented with

If you have installed the client you should be able to run it either by typing ```cfclient``` or ```python -m cfclient.gui``` in the console.

If it does not work, please tell us what instruction you followed and that error message you are getting.
